inc/wpseo-non-ajax-functions.php

```diff
@@ -385,8 +385,10 @@ function wpseo_adminbar_score( $score ) {
  * Enqueue CSS to format the Yoast SEO adminbar item.
  */
 function wpseo_admin_bar_style() {
+	
+	$options = WPSEO_Options::get_options( array( 'wpseo', 'wpseo_ms' ) );
```

Comment:
The option you are using to check against is located in the general `wpseo` option.
There does not seem to be a need to also request the `wpseo_ms` option.

Was there a reason to also include this? If not, could you change the line to:
`$options = WPSEO_Options::get_option( 'wpseo' );`